src/pages/Components/FileUploader/code.md

```diff
@@ -2,40 +2,54 @@
 title: 'Code'
 order: 2
 ---
-
 ## Import
 
+Import the settings and the checkbox `scss` files.
+
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.file-uploader';
 ```
 
 ## Basic usage
+> Natively, to upload files we use an `<input type="file"/>`, but in Mozaic, we will customize it with a `<label>` that will act as a button.
 
-Apply the classes...
+Wrap a input[type="file"] and a label inside a div and apply the following classes :
 
 ```html
-<div class="class">Label</div>
+<div class="mc-fileuploader">
+    <input type="file" aria-label="Click to choose a file on your device" class="mc-fileuploader__input" id="import-file">
+    <label for="import-file" class="mc-fileuploader__label">file uploader</label>  
+</div>
 ```
+<preview path="src/pages/Components/FileUploader/previews/file-uploader"></preview>
 
-<preview path="src/pages/Components/FileUploader/previews/FileUploader"></preview>
+## Behaviors
+### Standard file uploader states :
 
-## Variations
+- `hover`
+- `focus` 
+- `disabled`
 
-use the same variations in the same order as the index page
+<preview path="src/pages/Components/FileUploader/previews/file-uploader-all-states"></preview>
 
-## Behaviors
+### ⭐️ Bonus states
+
+If you wish, you can **customize** the file addition and their states with **javascript**, it is **already designed** for you 🤘 :
 
-use the same Behaviors in the same order as the index page
+<preview path="src/pages/Components/FileUploader/previews/file-uploader-with-file"></preview>
 
-## Responsive behaviors
+## Variations
 
-use the same Behaviors in the same order as the index page
+### Multiple files
+You can choose to upload **several files at the same time** with the multiple attribute.
+<preview path="src/pages/Components/FileUploader/previews/file-uploader-multiple"></preview>
 
-## Extension and customization
+### Accept type of files
```

Comment:
It's more a validation pattern instead of a variation. 
When you are using a input upload you have to declare what kind of file you accept without worrying about the variation

Reply:
it's a good point, i approve it